sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -44,7 +44,8 @@ fun HomeScreen(
     HomeScreenContent(
         uiModels = uiModels,
         showLoading = showLoading,
-        onItemClick = viewModel::navigateToSecond
+        onItemClick = viewModel::navigateToSecond,
+        onItemEdit = viewModel::navigateToThird
```

Comment:
I suggest renaming the callback a bit for consistency and to make more sense. We could have the callback with the name of the event used to trigger correctly; furthermore, when we change the handler, we don't need to reflect the name, for example we change the long click event to create a new item or remove it, in that case we have to change `onItemEdit` -> `onItemRemove` ...

```suggestion
        onItemLongClick = viewModel::navigateToThird
```